src/frontend/screens/Settings/components/LogSettings/index.tsx

```diff
@@ -98,7 +102,11 @@ export default function LogSettings({ isDefault, appName }: LogSettingsProps) {
   }, [isDefault, defaultLast])
 
   function showLogFileInFolder() {
-    window.api.showLogFileInFolder({ isDefault, appName })
+    window.api.showLogFileInFolder({
+      isDefault,
+      appName: isDefault ? '' : appName,
+      defaultLast
+    })
```

Comment:
Same here. Two arguments. `isDefault` is to much:
https://github.com/BrettCleary/HeroicGamesLauncher/blob/2d2e15b4dfa6c473a1e67e070e6a9e8822f14dbd/src/backend/logger/ipc_handler.ts#L12-L14